scripts/bootstrap

```diff
@@ -8,3 +8,27 @@ docker-compose run --entrypoint=npm client install
 docker-compose run --entrypoint=bower client install --allow-root
 # Initialize database
 docker-compose run server rake db:setup
+# Boot app
+docker-compose start
+
+# Get docker-machine ip if there is one
+port="$(docker-compose port nginx 80 | cut -d: -f2)"
+[ -z "$DOCKER_MACHINE_NAME" ] && host='localhost' || host="$(docker-machine ip)"
```

Comment:
![screen shot 2016-04-19 at 11 24 35 am](https://cloud.githubusercontent.com/assets/29782/14625821/51114882-0621-11e6-8847-f414fdb7f184.png)

```
$ docker-machine --version
docker-machine version 0.5.5, build 02c4254
```

Since 0.6, if no argument is provided, the `default` machine is used.